ui/src/tempVars/utils/index.ts

```diff
@@ -163,3 +167,43 @@ export const getLocalSelectedValue = (template: Template): string | null => {
 
   return null
 }
+
+export const csvToMap = (csv: string, onInvalidMapType: () => void) => {
+  const trimmed = _.trimEnd(csv, '\n')
+  const parsedTVS = Papa.parse(trimmed)
+  const templateValuesData = getDeep<string[][]>(parsedTVS, 'data', [[]])
```

Comment:
_nit_ Not sure the `getDeep` wrapper is necessary here (or anywhere):
```suggestion
  const templateValuesData: string[][] = _.get(parsedTVS, 'data', [[]])
```